dids/src/main/kotlin/web5/sdk/dids/methods/dht/DhtClient.kt

```diff
@@ -81,6 +81,7 @@ internal class DhtClient(
    * @throws PkarrRecordNotFoundException if the record is not found.
    * @throws PkarrRecordResponseException if the response from the dht gateway is not successful.
    */
+  @OptIn(ExperimentalStdlibApi::class)
```

Comment:
what's this needed for?

Reply:
had it in here to capture the hex response from the problem DID
can remove now